libs/books/feature/src/lib/snack-bar/snack-bar.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="snack-bar-header">
```

Comment:
Why not just use the default snack bar design? You didn't have to create an entire component for this.

Reply:
removing the custom snackbar component and showing a snackbar directly.